src/config.hpp

```diff
@@ -15,11 +16,13 @@ using color = const uint32_t;
 /// Actual window width will be equal to dxp_width + (dxp_padding * 2)
 ///
 const uint dxp_width = 0;
-const uint dxp_height = 150;
+const uint dxp_height = 250;
```

Comment:
Restore defaults pls

Reply:
Why tho? They'll be restored automatically when (if) we merge the branch, won't they?